computeengine/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "computeengine",
+  "description": "Collection of Node.js samples on Google Compute Engine.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache Version 2.0",
+  "author": "Google Inc.",
+  "engines": {
+    "node": ">=0.10.x"
+  },
+  "scripts": {
+    "sendgrid": "node sendgrid.js",
```

Comment:
nothing in this sample is really compute engine specific - at what point do we just have a single sendgrid sample, and use that for g[cka]e?